multiscale_phate/multiscale_phate.py

```diff
@@ -60,21 +75,39 @@ def __init__(
         self.gamma = gamma
         self.knn = knn
         self.n_jobs = n_jobs
+        self.NxTs = None
+        self.Xs = None
+        self.Ks = None
+        self.merges = None
+        self.Ps = None
+        self.diff_op = None
+        self.data_pca = None
+        self.pca_op = None
+        self.partition_clusters = None
+        self.dp_pca = None
+        self.epsilon = None
+        self.merge_threshold = None
+        self.gradient = None
+        self.levels = None
+
+
         super().__init__()
 
     def fit(self, X):
-        """Short summary.
+        """Builds a Diffusion Condensation tree to the data and computes
+        ideal levels of granularity via gradient analysis.
 
         Parameters
         ----------
-        X : type
-            Description of parameter `X`.
+        X : array, shape=[n_samples, n_features]
+            input data with `n_samples` samples and `n_dimensions`
+            dimensions. Accepted data types: `numpy.ndarray`,
+            `scipy.sparse.spmatrix` and `pd.DataFrame`.
 
         Returns
         -------
-        type
-            Description of returned object.
-
+       multiscale_phate_operator : Multiscale_PHATE
+        The estimator object
```

Comment:
Whitespace should be indented by 4

Reply:
This is still not indented correctly -- `The estimator object` should be indented by 4 more spaces.